services/hanke-service/src/integrationTest/resources/clear-db.sql

```diff
@@ -28,3 +28,4 @@ TRUNCATE TABLE
     ui_notification_banner,
     valmistumisilmoitus;
 UPDATE allu_status SET history_last_updated = '2017-01-01T00:00:00Z';
+ALTER SEQUENCE hankealue_id_seq RESTART WITH 1;
```

Comment:
Assuming certain entities get certain IDs leads to fragile tests. There should be better way to handle this.

Reply:
The test breaks, because it assumes the hankealue id in the database is what `ApplicationFactory.createExcavationNotificationArea()` uses as its default.

Create the hakemus first, find out the hankealue id and then give it as a parameter to the factory when creating the hakemusalue?

Reply:
Thank you for the suggestion, I did just that.